connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -158,6 +157,33 @@ public void testAdvertisedUri() {
         Assert.assertEquals("http://my-hostname:8080/", server.advertisedUrl().toString());
     }
 
+    @Test
+    public void testWadlOutput() {
```

Comment:
Nit: should the method be named `testOptionsDoesNotIncludeWadlOutput()` instead? The point of this PR is to prevent including WADL in the OPTIONS output, but the existing method name makes it seem like we're testing the content of the WADL output.

Reply:
Good suggestion, @rhauch , updated.